satpy/readers/ahi_hsd.py

```diff
@@ -234,7 +234,7 @@ class AHIHSDFileHandler(BaseFileHandler):
     masking of non-Earth pixels.
 
     In order to change the default behaviour, use the 'mask_space' variable
-    as part of ``reader_kwargs`` upon Scene creation::
+    as part of ``reader_kwargs`` upon Scene creation:
```

Comment:
I think this double colon is needed to render the code properly with sphinx. You can test this (with the appropriate sphinx dependencies installed) by running `make html` from the `doc` directory and opening `build/html/index.html`.

Reply:
Fixed in new commit